### PR TITLE
Addition of the 2017-11-02 Meeting 

### PR DESCRIPTION
**Pull request by ivan98q**
This is the Bitcoin Core meeting for 2017-11-02, sorry for taking so long the rest will be here after my finals are completed.

### CODE REVIEW

File: _posts/en/meetings/2017-11-02-meeting.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+## Bitcoin Core 0.15.1
+
+### Background
+
+  Moving to release the latest version of Bitcoin Core and what PRs need to be added to the backports.
+
+### Meeting Comments
+
+- Version was changed from0.15.0.2 to 0.15.1 for this release.
```
Comment (harding) [inline _posts/en/meetings/2017-11-02-meeting.md]: Missing space in "from0.15.0.2"

**Comment by achow101** [comment]
ACK modulo typo

File: _posts/en/meetings/2017-11-02-meeting.md (hunk 1 above)
Comment (ivan98q) [reply _posts/en/meetings/2017-11-02-meeting.md]: Thanks for catching that!

**Comment by MarcoFalke** [comment]
ACK

**Pull request merged**